src/Symfony/Component/Serializer/Mapping/Loader/XmlFileLoader.php

```diff
@@ -75,6 +70,16 @@ public function loadClassMetadata(ClassMetadataInterface $classMetadata)
     }
 
     /**
+     * Return the names of the classes mapped in this file.
+     *
+     * @return string[] The classes names
+     */
+    public function getMappedClasses()
```

Comment:
I don't think so, it may be useful in userland code. But if you prefer we can wait that someone of the core team gives his opinion :)